core/src/main/java/org/infinispan/persistence/manager/PersistenceManager.java

```diff
@@ -193,7 +193,9 @@
 
    /**
     * Returns the store one configured with fetch persistent state, or null if none exist.
+    * @deprecated Since 10.0, no longer used
     */
+   @Deprecated
```

Comment:
Yay! :)

Reply:
Actually looking again, it looks like we can just remove this method as the package is private.